src/critnib/critnib.c

```diff
@@ -58,6 +58,7 @@
 #include <stddef.h>
 
 #include "critnib.h"
+#include "utils_common.h"
```

Comment:
You added include but looks like do not use anything in this src file.

Reply:
I moved some misc helpers from concurrency.h to common.h that critnib uses.